OPENMEETINGS-2727 Add tooltips to whiteboard tab bar buttons and only  show scroll button when there is more then 1 whiteboard to scroll

sebawagner opened the pull request:
Add tooltip to buttons:
![image](https://user-images.githubusercontent.com/5152064/165007183-11e85591-f441-4e1a-b1d7-a5b84c6134b8.png)

And scroll button get only enabled in case there is more then 1 whiteboard available
![image](https://user-images.githubusercontent.com/5152064/165007262-c346fe94-d94e-4704-822a-74ad9a746b9f.png)

solomax reviewed (changes requested):
new terms are missing in `https://poeditor.com/projects/view_terms?id=333773&per_page=20&search=wb.` (which is now `source of truth` for labels)

Let me add new labels and re-generate label files
will take couple of minutes :)

then please resolve conflicts
sorry for extra work :(

solomax commented on openmeetings-web/src/main/java/org/apache/openmeetings/web/app/Application_de.properties.xml:
```diff
@@ -981,4 +981,7 @@ Bitte am OM server {3} anmelden und sie unter Admininistration -> Gruppen prüfe
 	<entry key="wizard.button.finish"><![CDATA[Ende]]></entry>
 	<entry key="zoom.FULL_FIT"><![CDATA[Vollformat]]></entry>
 	<entry key="zoom.PAGE_WIDTH"><![CDATA[Seitenbreite]]></entry>
+	<entry key="wb.add.whiteboard"><![CDATA[Add new whiteboard to conference room]]></entry>
```
German is missing :(

sebawagner replied:
You mean its not translated ? Cause the strings are there.

solomax replied:
yes, not translated :((

solomax commented:
@sebawagner new labels are added to PoEditor c22ab0ee3
Please update this PR and merge :)

ali-alhaidary commented:
And translated

sebawagner commented:
Thanks @ali-alhaidar! 
Sry do I need to re-import the language files from POEditor into our source code repo?

ali-alhaidary commented:
yes @sebawagner please.

solomax commented:
@sebawagner @ali-alhaidary I'll do re-import before release
Or by request :)

sebawagner commented:
I've updated the German transation (I think) 

How and where can you export the XML language files in POEditor ? I don't seem to be able to have this options

The instructions here don't seem to cover that case: https://openmeetings.apache.org/Internationalisation.html

I was able to join the project in POEditor: https://poeditor.com/join/project/6UF8Mhql61
But the link to "view" doesn't work for me https://poeditor.com/projects/view?id=333773
![image](https://user-images.githubusercontent.com/5152064/165231777-89ee45b4-eb21-4535-a9b3-dd552673b22c.png)


How can I export the files from POEditor ?

solomax commented:
I'm using this project to perform export https://github.com/solomax/om-poeditor
This link for view https://poeditor.com/projects/view?id=333773 works for me

You seems to be listed as contributor
I see no way so far how to add you as an admin :(

sebawagner commented:
Trying ... 
mvn exec:java -Dexec.args="~/.token ${OM_SOURCES}/" 

But your project fails to run any command:
```
[ERROR] Failed to execute goal on project om-poeditor: Could not resolve dependencies for project org.apache.solomax:om-poeditor:jar:1.0-SNAPSHOT: Could not find artifact org.apache.openmeetings:openmeetings-db:jar:6.0.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

I changed it then to 7.0.0-SNAPSHOT, which I have in my local ~/.m2 repo since I build this version locally before on the same machine.

But then the new error is:
```
INFO  04-26 19:11:38.498 o.a.solomax.App:221 [omax.App.main()] - Specific language can be specified after 'put', for ex. 'ar'
[WARNING] 
java.nio.file.NoSuchFileException: ~/.token
```

But there is a token:
```
sebastianwagner@Sebastians-MacBook-Pro om-poeditor % ls -alh ~/.token 
-rw-r--r--  1 sebastianwagner  staff    33B 26 Apr 19:04 /Users/sebastianwagner/.token
```

So I changed the command to:
```
mvn exec:java -Dexec.args="/Users/sebastianwagner/.token ${OM_SOURCES}/"
```

Then the command works but the POEditor response is then the error is:
```
INFO  04-26 19:19:50.885 o.a.c.s.W.RESP_IN:84 [omax.App.main()] - RESP_IN
    Address: https://api.poeditor.com/v2/projects/export
    Content-Type: application/json; charset=utf-8
    ResponseCode: 200
    ExchangeId: 70e3fef4-1b6b-4607-ba5b-a2387b223e86
    Headers: {Strict-Transport-Security=max-age=31536000; includeSubDomains; preload, Server=Apache/2.4.29 (Ubuntu), content-type=application/json; charset=utf-8, Content-Length=105, Date=Tue, 26 Apr 2022 07:19:50 GMT}
    Payload: {"response":{"status":"fail","code":"403","message":"You don't have permission to access this resource"}}

ERROR 04-26 19:19:50.898 o.a.solomax.App:93 [omax.App.main()] - call to projects/export was unsuccessful {"response":{"status":"fail","code":"403","message":"You don't have permission to access this resource"}}
```

Which makes sense cause my API Token is just contributor. I can't export the files via the API

Can you add me as an admin in POEditor:
https://poeditor.com/kb/how-to-add-an-administrator ?

Also we could just setup a Jenkins job to run the job and then create a zip file to download. However the dependency on openmeetings-db-x-x-x.jar will likely fail on Jenkins.

solomax commented:
well
these dependencies are required at build time only
I've made this helper looong time ago :)

Your Mac seems to unable resolve `~` to home dir
This works as expected at Linux :)

Thanks for the link!
You are admin now :)

sebawagner commented:
Will give a try. Why is there a 30sec Thread.sleep after each language file when it fetches it :D 
Thanks will open a new PR for the language changes.

ali-alhaidary commented:
@solomax @sebawagner all done in Arabic :-)

solomax commented:
@ali-alhaidary merged :)
